source/_drafts/Feature-Spotlight-Accessibility.md

```diff
@@ -0,0 +1,103 @@
+---
+title: 'Feature Spotlight: Accessibility'
+author:
+  name: Brandon Casey
+  github: brandonocasey
+tags:
+  - video.js 6.0
+  - accessibility
+  - a11y
+---
+
+Accessibility! The most important feature you never knew about.
+
+In the Video.js organization we try hard to have good accessiblity. Like most other software any change can effect the system in unintended ways. For example `MuteToggle` and `VolumeControl` were recently (in video.js 5) married into `VolumeMenuButton`. While this change did allow these controls to work in tandem. It also did something unintended. It broke accessibility. In this post we will go over what broke, what the fix was, what accessibility is, and how to test and make sure it works.
```

Comment:
- `effect` should be `affect`
- `video.js` should be `Video.js`
- `work in tandem. It also` should be `work in tandem, it also`